docs/installation/centos.md

```diff
@@ -44,45 +44,7 @@ reported kernel bugs may have already been fixed on the latest kernel packages
 
 ## Install
 
-You use the same installation procedure for all versions of CentOS,
-only the package you install differs. There are two packages to choose from:
-
-<table>
-  <tr>
-    <th>Version</th>
-    <th>Package name</th>
-  </tr>
-  <tr>
-    <td>6.5 and  higher</td>
-    <td>
-    <p>
-    <a href="https://get.docker.com/rpm/1.7.1/centos-6/RPMS/x86_64/docker-engine-1.7.1-1.el6.x86_64.rpm">
-    https://get.docker.com/rpm/1.7.1/centos-6/RPMS/x86_64/docker-engine-1.7.1-1.el6.x86_64.rpm</a>
-    <p>
-    <a href="https://get.docker.com/rpm/1.7.1/centos-6/SRPMS/docker-engine-1.7.1-1.el6.src.rpm">
-   https://get.docker.com/rpm/1.7.1/centos-6/SRPMS/docker-engine-1.7.1-1.el6.src.rpm</a>
-    <p>
-    </p>
-    </td>
-  </tr>
-  <tr>
-    <td>7.X</td>
-    <td>
-    <p>
-     <a href="https://get.docker.com/rpm/1.7.1/centos-7/RPMS/x86_64/docker-engine-1.7.1-1.el7.centos.x86_64.rpm">
-    https://get.docker.com/rpm/1.7.1/centos-7/RPMS/x86_64/docker-engine-1.7.1-1.el7.centos.x86_64.rpm</a>   
-    </p>
-    <p>
-     <a href="https://get.docker.com/rpm/1.7.1/centos-7/SRPMS/docker-engine-1.7.1-1.el7.centos.src.rpm">
-    https://get.docker.com/rpm/1.7.1/centos-7/SRPMS/docker-engine-1.7.1-1.el7.centos.src.rpm</a>   
-    </p>
-    </td>
-  </tr>
-</table>
-
-
-This procedure depicts an installation on version 6.5. If you are installing on
-7.X, substitute that package for your installation. 
+You use the same installation procedure for all versions of CentOS.
```

Comment:
@jfrazelle  Thanks for taking an initial pass. 

This first sentence isn't actually true any more, it is only true if a user is using the installation script.  If the user is not using the script, the procedure requires different commands.   So, I've rewritten this one to illustrate.  The others need to use the same pattern.

## Install Docker Engine

There are two ways to install Docker Engine.  You can use `curl` with the  `get.docker.com` site. This method runs an installation script which installs via the `yum` package manager. Or you can install with the `yum` package manager directly yourself.

### Install with the script

You use the same installation procedure for all versions of CentOS.

1. Log into your machine as a user with `sudo` or `root` privileges.

2. Make sure your existing yum packages are up-to-date.

		$ sudo yum update
		
3. Run the Docker installation script.
		
		$ curl -sSL https://get.docker.com/ | sh

4. Start the Docker daemon.

		$ sudo service docker start

5. Verify `docker` is installed correctly by running a test image in a container.

		$ sudo docker run hello-world
		Unable to find image 'hello-world:latest' locally
		latest: Pulling from hello-world
		a8219747be10: Pull complete 
		91c95931e552: Already exists 
		hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
		Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd1.7.1cf5daeb82aab55838d
		Status: Downloaded newer image for hello-world:latest
		Hello from Docker.
		This message shows that your installation appears to be working correctly.

		To generate this message, Docker took the following steps:
		 1. The Docker client contacted the Docker daemon.
		 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
				(Assuming it was not already locally available.)
		 3. The Docker daemon created a new container from that image which runs the
				executable that produces the output you are currently reading.
		 4. The Docker daemon streamed that output to the Docker client, which sent it
				to your terminal.

		To try something more ambitious, you can run an Ubuntu container with:
		 $ docker run -it ubuntu bash

		For more examples and ideas, visit:
		 http://docs.docker.com/userguide/
		 
## Install without the script

1. Log into your machine as a user with `sudo` or `root` privileges.

2. Make sure your existing yum packages are up-to-date.

		$ sudo yum update
    
3. Add the yum repo yourself.

    For CentOS 6 run:

		$ cat >/etc/yum.repos.d/docker.repo <<-EOF
        [dockerrepo]
        name=Docker Repository
        baseurl=https://yum.dockerproject.org/repo/main/centos/6
        enabled=1
        gpgcheck=1
        gpgkey=https://yum.dockerproject.org/gpg
        EOF

    For CentOS 7 run:

		$ cat >/etc/yum.repos.d/docker.repo <<-EOF
        [dockerrepo]
        name=Docker Repository
        baseurl=https://yum.dockerproject.org/repo/main/centos/7
        enabled=1
        gpgcheck=1
        gpgkey=https://yum.dockerproject.org/gpg
        EOF

4. Install the Docker package.

        $ yum install docker-engine
        
4. Start the Docker daemon.

		$ sudo service docker start

5. Verify `docker` is installed correctly by running a test image in a container.

		$ sudo docker run hello-world